src/Microsoft.OData.Edm/Vocabularies/Annotations/IEdmTermExtensions.cs

```diff
@@ -43,12 +31,28 @@ public static IEdmVocabularyAnnotation CreateVocabularyAnnotation(this IEdmTerm
                 throw new InvalidOperationException(Strings.EdmVocabularyAnnotations_DidNotFindDefaultValue(term.Type));
             }
 
-            return new EdmVocabularyAnnotation(target, term, qualifier, value)

```

Comment:
why did we get rid of the overload that takes a qualifier?

Reply:
It's because we were passing null to it and constructor call also passes null : `        public EdmVocabularyAnnotation(IEdmVocabularyAnnotatable target, IEdmTerm term, IEdmExpression value)
            : this(target, term, null, value)`

Reply:
It will be different though if someone calls the extension overload that takes the `qualifier`. I'll add it back.